spec/controllers/registrations_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe 'Sign Up user via POST', type: :request do
```

Comment:
Rails is convention over configuration, so that I recommend to split this file. One for each method and with the same file structure as the controller.

For example, for sign up should be `spec/controllers/api/v1/registrations/create_spec.rb`

Reply:
Noted! Will do. Thanks